src/common/transformations/include/transformations/op_conversions/fake_convert_decomposition.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/matcher_pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API FakeConvertDecomposition;
+
+}  // namespace pass
+}  // namespace ov
+
+/**
+ * @ingroup ov_transformation_common_api
+ * @brief FakeConvertDecomposition transformation decomposes FakeConvert layer.
+ * f8: f8e4m3, f8e5m2
+ * downconvert: f32->f8, f16->f8, bf16->f8
+ * upconvert: f8->f32, f8->f16, f8->bf16
```

Comment:
Only these precision are supported by decomposition transformation or this is just an example?

Reply:
Thanks Dmitry for the comment! Yes, they are the only supported precisions, which are assured by https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/op/fake_convert.cpp#L90-L110 and https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/op/fake_convert.cpp#L129-L134.